Jenkinsfile_nightly

```diff
@@ -4,7 +4,7 @@ properties([
   // H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
   pipelineTriggers([cron('H 05 * * *')]),
   parameters([
-    string(name: 'URL_TO_TEST', defaultValue: 'http://wa-task-management-api-aat.service.core-compute-aat.internal', description: 'The URL you want to run these tests against'),
+    string(name: 'URL_TO_TEST', defaultValue: 'http://wa-workflow-api-aat.service.core-compute-aat.internal', description: 'The URL you want to run these tests against'),
```

Comment:
We can clean up those parameters since the nightly pipeline does not run functional tests and only runs mutation tests.